packages/create-svelte/templates/default/src/routes/todos/index.ts

```diff
@@ -1,6 +1,7 @@
 import { api } from './_api';
-import type { RequestHandler } from '@sveltejs/kit';
+import type { RequestHandler } from './index';
```

Comment:
Why RequestHandler is imported from ./index instead of @sveltejs/kit

Reply:
For those sweet sweet [generated types](https://kit.svelte.dev/docs/types#generated-types)

Reply:
<img width="543" alt="image" src="https://user-images.githubusercontent.com/42126168/166685712-dd00bc89-ea03-45f8-af53-7e0eebf4e3d5.png">
<img width="504" alt="image" src="https://user-images.githubusercontent.com/42126168/166685854-01a5c5e0-8bc0-4a8b-975b-01ef15656224.png">
this confuses me

Reply:
See #4705

Reply:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/42126168/166687336-0b0568dd-6293-48a9-ad55-f83ff1bb38a5.png">
Correct parsing, maybe it's time to merge 4705